libraries/monitor/src/plugins/sentry.js

```diff
@@ -0,0 +1,24 @@
+const Sentry = require('@sentry/node');
+
+class SentryReporter {
+  constructor({dsn, taskclusterVersion, serviceName, processName}) {
```

Comment:
For ease of deployers figuring out what's wrong, it might be nice to assert that dsn is set.